syntax = "proto3";

option
java_package = "com.huasheng.quant.open.sdk.protobuf.common.response";
option java_outer_classname = "CommonStringResponseProto";
message CommonStringResponse {
	string data = 1;
}